src/test/kotlin/no/nav/tilleggsstonader/sak/vedtak/boutgifter/beregning/BoutgifterBeregningMidlertidigUtgiftTest.kt

```diff
@@ -183,7 +183,7 @@ class BoutgifterBeregningMidlertidigUtgiftTest {
                 vedtaksperioder = vedtaksperioder,
                 typeVedtak = TypeVedtak.INNVILGELSE,
             )
-        }.hasMessageContaining("Vi støtter foreløpig ikke at utgifter krysser ulike utbetalingsperioder")
+        }.hasMessageContaining("Utgiftsperioder krysser beregningsperioder")
```

Comment:
Siden denne feilmeldingen nå har ganske mye "logikk" i seg burde vi kanskje asserte på hele feilmeldingen i stedet for bare en liten del av den. Forslag:
```suggestion
                }.hasMessage(
            """
            Utgiftsperioder krysser beregningsperioder

            Utgiftsperiode 25.01.2025–05.02.2025 krysser beregningsperiodene: 
             - 01.01.2025–31.01.2025
             - 01.02.2025–28.02.2025
            
            Utgiftsperioden(e) må splittes.
        """.trimIndent()
        )
```

Reply:
Jeg la til hele feilmeldingen her: https://github.com/navikt/tilleggsstonader-sak/pull/737/commits/eacc8e3110c90e7ac63849a86d21a8c5e05a6a97